app/javascript/packs/notificationSubscriptionHandler.js

```diff
@@ -1,100 +0,0 @@
-/* global showLoginModal */
```

Comment:
https://forem-team.slack.com/archives/C7GE84DEJ/p1663311391923539 

it seems like this file is not being used anywhere.